boards/arm/adafruit_feather_m0_lora/doc/index.rst

```diff
@@ -0,0 +1,180 @@
+
+The default configuration can be found in the Kconfig file
+:zephyr_file:`boards/arm/adafruit_feather_m0_lora/adafruit_feather_m0_lora_defconfig`.
+
+Connections and IOs
+===================
+
+The `Adafruit Feather M0 with LoRa radio module Learn site`_ has detailed
+information about the Basic Proto board variant including `pinouts`_
+and the `schematic`_.
+
+System Clock
+============
+
+The SAMD21 MCU is configured to use the 32.768 kHz external oscillator
+with the on-chip PLL generating the 48 MHz system clock.
+
+Serial Port
+===========
+
+The SAMD21 MCU has 6 SERCOM based USARTs.  On the Adafruit Feather M0
+Basic Proto, SERCOM0 is the Zephyr console and is available on pins 0
+(RX) and 1 (TX).
+
+I2C Port
+========
+
+The SAMD21 MCU has 6 SERCOM based USARTs.  On the Adafruit Feather M0
+Basic Proto, SERCOM3 is available on pin 20 (SDA) and pin 21 (SCL).
+
+SPI Port
+========
+
+The SAMD21 MCU has 6 SERCOM based SPIs.  On the Adafruit Feather M0
+Basic Proto, SERCOM4 is available on pin 22 (MISO), pin 23 (MOSI), and
+pin 24 (SCK).
+
+USB Device Port
+===============
+
+The SAMD21 MCU has a USB device port that can be used to communicate
+with a host PC.  See the :ref:`usb-samples` sample applications for
+more, such as the :ref:`usb_cdc-acm` sample which sets up a virtual
+serial port that echos characters back to the host PC.
+
+LoRa Radio
+==========
+The Semtech SX127x radio chip on the LoRa variant is attached to the
+SPI port (SERCOM4). Depending on the hardware version, 433MHz or
+900MHz is supported.
+
+Programming and Debugging
+*************************
+
+The Adafruit Feather M0 Basic Proto ships with a BOSSA compatible
+SAM-BA bootloader.  The bootloader can be entered by quickly tapping
+the reset button twice.
+
+Flashing
+========
+
+#. Build the Zephyr kernel and the :ref:`hello_world` sample application:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: adafruit_feather_m0_lora
+      :goals: build
+      :compact:
+
+#. Connect the Adafruit Feather M0 Basic Proto to your host computer
+   using USB
+
+#. Connect a 3.3 V USB to serial adapter to the board and to the
+   host.  See the `Serial Port`_ section above for the board's pin
+   connections.
+
+#. Run your favorite terminal program to listen for output. Under Linux the
+   terminal should be :code:`/dev/ttyACM0`. For example:
+
+   .. code-block:: console
+
+      $ minicom -D /dev/ttyACM0 -o
+
+   The -o option tells minicom not to send the modem initialization
+   string. Connection should be configured as follows:
+
+   - Speed: 115200
+   - Data: 8 bits
+   - Parity: None
+   - Stop bits: 1
+
+#. Tap the reset button twice quickly to enter bootloader mode
+
+#. Flash the image:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: adafruit_feather_m0_lora
+      :goals: flash
+      :compact:
+
+   You should see "Hello World! arm" in your terminal.
```

Comment:
arm?

Reply:
Yes, see https://github.com/zephyrproject-rtos/zephyr/blob/a61ca79041a0b8679a77a78228fff7698e2be1d2/samples/hello_world/src/main.c#L11

Reply:
Right, should be changed.
Same issue in upstream board https://github.com/zephyrproject-rtos/zephyr/blame/main/boards/arm/adafruit_feather_m0_basic_proto/doc/index.rst#L153

Reply:
And actually a generic issue since https://github.com/zephyrproject-rtos/zephyr/commit/e36c7f5b2bff88b2bc4302325a1e39c1265cce5b